dev

```diff
@@ -19,51 +20,22 @@ LDFLAGS+=" -X 'github.com/mesg-foundation/core/config.EnvMarketplaceToken=0x5861
 
 MESG_SERVER_PORT=${MESG_SERVER_PORT:-50052}
 
-# if set to true the docker with system services will be started.
-MESG_SERVICE_SERVER_ON=${MESG_SERVICE_SERVER_ON:-true}
-MESG_SERVICE_SERVER="engine-service-server"
-
-
 networkExists=$(docker network list -f name=engine -f driver=overlay -q)
 if [ "$networkExists" == "" ]; then
   docker network create --driver overlay engine --label com.docker.stack.namespace=engine
 fi
 
-function deleteServiceServer {
-  rm -rf tmp-systemservices
-  if [ "$MESG_SERVICE_SERVER_ON" = true ]; then
-    if docker service list | grep -q engine-service-server; then
-      docker service rm $MESG_SERVICE_SERVER
-    fi
-  fi
-}
-
-deleteServiceServer
-mkdir -p tmp-systemservices
-for i in systemservices/* ; do
-  if [ -d "$i" ]; then
-    echo "Creating archive for $i..."
-    cd $i
-    file=$(basename "$i")
-    tar -czf ../../tmp-systemservices/$file.tar.gz --exclude-from=.dockerignore .
-    varname="$(tr '[:lower:]' '[:upper:]' <<< ${file:0:1})${file:1}"
-    LDFLAGS+=" -X 'github.com/mesg-foundation/core/config.${varname}URL=http://$MESG_SERVICE_SERVER/$file.tar.gz'"
-    cd ../..
+# from scripts/build-core.sh
+# upload system services
+for s in systemservices/* ; do
+  if [ -d "$s" ]; then
+    pushd $s > /dev/null
+    name=$(basename "$s")
+    varname="${name}"
+    LDFLAGS+=" -X 'github.com/mesg-foundation/core/config.${varname}Compiled=$(cat compiled.json)'"
```

Comment:
```suggestion
    LDFLAGS+=" -X 'github.com/mesg-foundation/engine/config.${varname}Compiled=$(cat compiled.json)'"
```

Reply:
where the compiled.json is from?

Reply:
also, why not compiling the service also in this script? like scripts/build-engine.sh is doing? optimization?